content/ar/docs/tutorials/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: برامج تعليمية
+main_menu: true
+weight: 60
```

Comment:
```suggestion
main_menu: true
no_list: true
weight: 60
```

Please ensure that most parameters in the header match those in the English document to maintain synced. 😊

For reference, you can view the English document's header at:
https://raw.githubusercontent.com/kubernetes/website/dev-1.29-ar.1/content/en/docs/tutorials/_index.md